sass/directives/_button.scss

```diff
@@ -218,3 +218,49 @@
   font-size: $massive-button-font-size;
   padding: $massive-button-vertical-padding $massive-button-horizontal-padding;
 }
+
+// Accordion Buttons
+@mixin accordion {
```

Comment:
Another thing I can take care of, but I'd name this like the other buttons in this file - i.e. `accordion-button`. Additionally, I 'd collapse this function with the rest of the accordion behavior in its own file, since it's really not used as a "major button" across the site - it's only used with in the context of a single feature - an accordion